### ISSUE TITLE
Titles with an apostrophe completely break the /agentinvite page.

### ISSUE THREAD

**Issue by MailYouLater** [Ylianst/MeshCentral#2664]
If you set your server's `title` to something that includes an apostrophe then opening an invite link will load a blank page.  This is because the page starts with the `<body>` tag set to `display: none`, which is removed by some JavaScript later, but on this page the `title` is used in some JavaScript without escaping non-safe characters, thus the apostrophe in the `title` causes an error in the JavaScript and causes it to fail before it would set the `<body>` tag to be visible.

Here's an example config line with an offending apostrophe.
```
      "title": "Company's MeshCentral Instance",
```
This line would be used where this example is found in the sample config:
https://github.com/Ylianst/MeshCentral/blob/e3e810bd8fa703781b8e0c5ebc4c9a974c797147/sample-config.json#L17

The appropriate fix is to remove one set of curly braces from around `title` on this line.  
https://github.com/Ylianst/MeshCentral/blob/e3e810bd8fa703781b8e0c5ebc4c9a974c797147/views/agentinvite.handlebars#L165
i.e.:
```diff
-         var title = '{{{title}}}';
+         var title = '{{title}}';
```

**Comment by MailYouLater** [comment]
As simple as this fix is, I've noticed that the "triple-stash" (as the handlebars documentation calls it) is used a lot in the MeshCentral source.  Seeing as handlebars' documentation seems to indicate that two sets of curly braces is intended to be the common usage, and that 'triple-stash' usage is intended for specific instances when you know you don't want handlebars to sanitize your strings for you, and considering that sanitizing strings by default is generally intended to improve security and stability... I wonder if it might be a good idea to remove the 'extra' third set of curly braces from the parts of MeshCentral that don't require it...?

**Comment by Ylianst** [comment]
Oh! Thanks for reporting this. Working on this now.

**Comment by Ylianst** [comment]
Just checked in a fix for this. Will be in MeshCentral v0.8.42. Thanks for reporting.

Also, I need to look at "triple-stash". Indeed, I may not be using it correctly. In general, I URL escape any strings that could cause issues but I could be doing this all wrong.

**Comment by Ylianst** [comment]
Just published MeshCentral v0.8.42 that includes this fix. (Also, MeshCentral v0.8.42 has MeshCentral Router and MeshCentral Assistant code signed).

**Comment by MailYouLater** [comment]
Confirmed.  Your method may have been different... IDK which is _better_ but v0.8.42 does fix the issue.
